### PR TITLE
Added feature flag

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#5471

*Description of changes:*
- incremented app version
- added LD client SDK package
- added LD config key
- init LD at app start
- check FF at app start
- added FF plugin
- load LD key from config

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/public/config/configuration.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "BUSINESSES_URL": "https://dev.bcregistry.ca/business/auth/",
     "KEYCLOAK_CONFIG_PATH": "/local-keycloak-config-url/keycloak.json",
     "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",
-    "LEGAL_API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1"
+    "LEGAL_API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1",
+    "LD_CLIENT_ID": ""
```
Comment (severinbeauvais) [inline client/public/config/configuration.json:9]: LD is disabled when running locally.

Comment (severinbeauvais) [reply client/public/config/configuration.json:9]: They keys for Dev/Test/Prod environments need to be set for LD to work. See #5732.

**Review by cameron-freshworks — APPROVED** [review]
LGTM!

**Review by cameron-freshworks — APPROVED** [review]
LGTM!

**Review by scottrumsby — APPROVED** [review]
i have nothing to add here

**Pull request merged**